### ISSUE TITLE
Config error

### ISSUE THREAD

**Issue by ikke68** [pippyn/Home-Assistant-Sensor-Afvalbeheer#40]
Hi,

I'm trying to set up this plugin in HA i have hacs installed and moved the files to it's location, but i get a syntax error from HA `Error loading /config/configuration.yaml: while parsing a block mapping
  in "/config/sensors.yaml", line 2, column 7
expected <block end>, but found '<scalar>'
  in "/config/sensors.yaml", line 12, column 40` 
i copied your code and only changed my zipcode and number, `  sensor:
    - platform: afvalbeheer
      wastecollector: Cure            (required)
      resources:                       (at least 1 required)
        - restafval
        - gft
        - papier
        - pmd
      postcode: 5645BE                 (required)
      streetnumber: 122                (required)
      upcomingsensor: 0                (optional)
      dateformat: '%d-%m-%Y'           (optional)
      dateonly: 0                      (optional)
      name: ''                         (optional)
      nameprefix: 1                    (optional)
      builtinicons: 0                  (optional)
      disableicons: 0                  (optional)
      dutch: 0                         (optional)`
I hope you guys can help me :)

have a awesome day,

**Comment by pippyn** [comment]
Sure love to help, but please format your code. This is unreadable.

**Comment by ikke68** [comment]
Hi,
`    - platform: afvalbeheer
      wastecollector: Cure            (required)
      resources:                       (at least 1 required)
        - restafval
        - gft
        - papier
      postcode: 5645BE                 (required)
      streetnumber: 122                (required)
      upcomingsensor: 0                (optional)
      dateformat: '%d-%m-%Y'           (optional)
      dateonly: 0                      (optional)
      name: ""                         (optional)
      nameprefix: 1                    (optional)
      builtinicons: 0                  (optional)
      dutch: 1                         (optional)
'

**Comment by ikke68** [comment]
This is the error from home assistant 
'

**Comment by pippyn** [comment]
I'm sorry but this way I can't see if the indentation is correct. But what I do see is that you need to remove the comments. Everything in the brackets and the brackets too. Like; (required) and (optional)

**Comment by ikke68** [comment]
Is a picture okay for you? From notepad++

**Comment by pippyn** [comment]
Well it should be something like this;
```
- platform: afvalbeheer
  wastecollector: Cure
  resources:
    - restafval
    - gft
    - papier
  postcode: 5645BE
  streetnumber: 122
  upcomingsensor: 1
  dateformat: '%d-%m-%Y'
  builtinicons: 1
  dutch: 1
```

**Comment by ikke68** [comment]
Hi,

It's working now but its showing the wrong pick up date...  it says papier pickup will be 20-02 but it needs to be 06-02, any idea how to fix this?

**Comment by pippyn** [comment]
You are right. Fixed in version v2.5.10
